finalPrep/modZeroHero.js

```diff
@@ -2,40 +2,80 @@
 
 // Declare two variables - heroName AND specialAbility - set to strings
 
+var heroName = "Super Janika"
+var specialAbility = "telekinesis"
+
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
+var greeting = ("What up playas it's " + heroName + " !");
+console.log(greeting);
+
+var catchphrase = (`Slamma blamma ding dong! Better watch out or I'll use ${specialAbility} on you!`);
+console.log(catchphrase);
+
 // Declare two variables - power AND energy - set to integers
 
+var power = 3000
+var energy = 51
+
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
 
+var fullPower = power * 500
+console.log(fullPower);
+
+var fullEnergy = energy + 150
+console.log(fullEnergy);
+
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
 
+var isHuman = false;
+var identityConcealed = true;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
 
+var archEnemies = ["The Plastics", "Trump", "Ronald McDonald"];
+var sideKicks = ["Opposums", "All the Dogs", "Kourtney Kardashian"];
+
 // Print the first sidekick to your console
 
+console.log(sideKicks[0]);
+
 // Print the last archEnemy to the console
 
+console.log(archEnemies[2]);
+
 // Write some code to add a new archEnemy to the archEnemies array
 
+archEnemies.push("The Blob");
+
 // Print the archEnemies array to console to ensure you added a new archEnemy
 
+console.log(archEnemies);
+
 // Remove the first sidekick from the sidekicks array
 
+sideKicks.shift();
+
 // Print the sidekicks array to console to ensure you added a new sidekick
 
+console.log(sideKicks);
+
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
 
+function assessSituation(){
+
```

Comment:
Please come back to this to add your function and class work. Reference some of my other comments to find a good example of functions that take parameters, and see what you can come up with.